<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>841</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Пересечение отрезков</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="generation">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>151</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Генерация отрезков</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>10</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="finish">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>31</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>&gt;|</string>
     </property>
    </widget>
    <widget class="QPushButton" name="stop">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>31</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>10</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="backsteps">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>31</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;&lt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="backstep">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>31</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="forwardstep">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>31</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="forwardsteps">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>20</y>
       <width>31</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>&gt;&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pause">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>31</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>||</string>
     </property>
    </widget>
   </widget>
   <widget class="Area" name="area" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>801</width>
      <height>361</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>801</width>
      <height>361</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="load">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Загрузка из файла</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>30</y>
      <width>97</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Сохранить</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>841</width>
     <height>25</height>
    </rect>
   </property>
<<<<<<< .merge_file_mJVoqF
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
=======
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveResult"/>
   </widget>
   <widget class="QMenu" name="menuGeneration">
    <property name="title">
     <string>Генерация отрезков</string>
    </property>
    <addaction name="actionRandom"/>
    <addaction name="actionDraw"/>
    <addaction name="actionStopDraw"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Правка</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuGeneration"/>
   <addaction name="menuEdit"/>
>>>>>>> .merge_file_GeSpaC
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
<<<<<<< .merge_file_mJVoqF
=======
  <action name="actionOpen">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Сохранить отрезки</string>
   </property>
  </action>
  <action name="actionRandom">
   <property name="text">
    <string>Случайная генерация</string>
   </property>
  </action>
  <action name="actionDraw">
   <property name="text">
    <string>Нарисовать отрезки</string>
   </property>
  </action>
  <action name="actionStopDraw">
   <property name="text">
    <string>Закончить рисование</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Отменить</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Повторить</string>
   </property>
  </action>
  <action name="actionSaveResult">
   <property name="text">
    <string>Сохранить результат</string>
   </property>
  </action>
  <action name="actionCompareResults">
   <property name="text">
    <string>Сравнить результаты</string>
   </property>
  </action>
>>>>>>> .merge_file_GeSpaC
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Area</class>
   <extends>QWidget</extends>
   <header location="global">area.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
